pages/common/gnucash-cli.md

```diff
@@ -0,0 +1,12 @@
+# gnucash-cli
+
+> A command-line version of GnuCash.
+> More information: <https://manned.org/gnucash-cli>.
+
+- Get online quotes for currencies and stocks in the file:
```

Comment:
```suggestion
- Save online quotes for currencies and stocks to a file:
```
Is this still correct?

Reply:
No, unfortunately. What the command would do is read the data file (`example.gnucash`) and get a list the foreign currencies and stocks in it. Then it would log online quotes for those currencies to the console.

Reply:
Now I get what the command does. What about this:
```suggestion
- Get quotes for currencies and stocks specified in a file and print them to stdout:
```